web/themes/new_weather_theme/templates/partials/satellite.html.twig

```diff
@@ -0,0 +1,22 @@
+{% if weather.satellite %}
+<div class="grid-container margin-top-2 padding-x-0 tablet:padding-x-2" wx-outer-radar-container>
+  <div class="grid-row tablet-lg:grid-col-8 tablet-lg:grid-offset-2">
+    <h3 class="wx-visual-h2 text-normal text-primary-darker padding-x-2 tablet:padding-x-0 margin-top-3 margin-bottom-2">{{ "Satellite" | t }}</h3>
+  </div>
+
+  <div class="grid-container padding-0 bg-white shadow-1 grid-col-12 tablet-lg:grid-offset-2 tablet-lg:grid-col-8">
+    <div class="usa-sr-only">
+      <p>
+        {{ "Listed below is an animated loop of the most recent
+            satellite imagery for @place."
+            | t({"@place": weather.place })
+        }}
+      </p>
+    </div>
+
+    <div aria-hidden="true">
+      <img src="{{ weather.satellite.gif }}" class="wx-minw-full">
```

Comment:
From  some light searching, it seems like even though it's aria-hidden we should have a blank `alt` attribute on this. 

I am wondering if there is a reason not to use alt text here instead of the sr-only div/paragraph above?